Makefile

```diff
@@ -0,0 +1,157 @@
+# Makefile for shipping and testing the container image.
+
+MAKEFLAGS += --warn-undefined-variables
+.DEFAULT_GOAL := build
+.PHONY: *
+
+# we get these from CI environment if available, otherwise from git
+GIT_COMMIT ?= $(shell git rev-parse --short HEAD)
+GIT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
+WORKSPACE ?= $(shell pwd)
+
+namespace ?= autopilotpattern
+tag := branch-$(shell basename $(GIT_BRANCH))
+image := $(namespace)/prometheus
+testImage := $(namespace)/prometheus-testrunner
+
+#dockerLocal := DOCKER_HOST= DOCKER_TLS_VERIFY= DOCKER_CERT_PATH= docker
```

Comment:
This does not work on my Mac with Docker Machine. Replacing it with `dockerLocal := docker` does.

Error is:

```
$ make build
DOCKER_HOST= DOCKER_TLS_VERIFY= DOCKER_CERT_PATH= docker build -f test/Dockerfile -t=autopilotpattern/prometheus-testrunner:branch-triton-support .
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
make: *** [test-runner] Error 1
```